core/src/components/input/input.tsx

```diff
@@ -272,6 +273,36 @@ export class Input implements ComponentInterface {
    */
   @Prop({ mutable: true }) value?: string | number | null = '';
 
+  /**
+   * The predefined format of the user's input. For example, you can set a mask
+   * that only accepts digits, or you can configure a more complex pattern like
+   * a phone number or credit card number.
+   *
+   * The mask supports two formats:
+   * 1. A valid [regular expression pattern](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions)
+   * 2. An array containing regular expression and fixed character patterns
+   *
+   * Each string in the mask array format is a fixed character in the mask.
```

Comment:
When I first read this I understood it to mean each entry in the array is a fixed character. This would contradict the above line which states the array can accept a regex. However, I now understand that it means string types.

Perhaps we can just say "The fixed characters in the mask cannot be erased or replaced by the user"